settings.gradle

```diff
@@ -1,4 +1,4 @@
 rootProject.name = 'terra-data-catalog'
-include('service', 'client', 'rawls-client', 'integration', 'scripts')
+include('admin-cli', 'service', 'client', 'rawls-client', 'library', 'integration', 'scripts')
```

Comment:
I called the subproject `library` but it occurs to me that it's not a library we build. Maybe it should be called `common` instead?

Reply:
I like `common` more than `library`, I think.

Reply:
I just realized we already have `common` where we put `openapi.yml` and `postgres-init.sql`. The sql could be moved to `scripts`, perhaps, and maybe `openapi.yml` gets its own directory. Or we could put it in `service` or `client` and have the other use a relative path to include it.

Reply:
I moved `Postgres-init.sql` in to `scripts`. I moved `openapi.yml` into `service/.../resources`